### DISCUSSION TITLE
Is it possible to search for checkboxes (task lists)?

### DISCUSSION THREAD

**Discussion by simicvm** [outline/outline#6684] [General] [answered]
Basically is it possible to search and find all existing tasks?

**Comment by tommoor** [comment, marked as answer]
Not currently, task lists would need to be parsed and indexed separately.

**Reply by simicvm** [reply]
Thanks for the info.

Could we just enable search with "[ ]" and "[x]"? Those are already the shortcuts for making them.

**Reply by tommoor** [reply]
Unfortunately not, special characters are stripped in the search index – this is a limitation of postgres full text indexing

**Reply by tommoor** [reply]
FWIW, I do think at some point we'll start storing these in the database – it's not a particularly hard task, but there are other more pressing things in the backlog :)
